### PR TITLE
Issue 1926 max

### PR DESCRIPTION
**Pull request by Maxstgt**

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <tunnel-layout [steps]="steps" exitRedirect="/c/o/dashboard">
   <header fxLayout="row" fxLayoutAlign="space-between center">
     <tunnel-step-stat></tunnel-step-stat>
-    <img asset="archipel-content.svg" type="logo" alt="Archipel Content Logo" />
+    <img asset="logo_archipel_content_primary.svg" type="logo" alt="Archipel Content Logo" />
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.html]: there is a logo "archipel-content-logo", verify that it's not the same. I don't want duplicate images in the assets.

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.html [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
-  <ng-container *ngIf="!loading else isLoading">
-    <a mat-icon-button [routerLink]="routeBeforeTunnel">
-      <mat-icon svgIcon="cross"></mat-icon>
-    </a>
-  </ng-container>
-  <ng-template #isLoading>
-    <mat-spinner diameter="50"></mat-spinner>
-  </ng-template>
+<mat-toolbar class="flex-container">
+  <img asset="logo_archipel_content_primary.svg" type="logo" alt="Archipel Content Logo" class="item-center" />
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.html]: Same as above

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.ts [hunk 3]
```diff
@@ -51,13 +44,13 @@ export class StartTunnelComponent implements OnInit {
     this.loading = true;
     try {
       const createdBy = this.auth.getValue().uid;
-      const movieId = await this.movieService.add({ _meta: { createdBy }});
+      const movie = createMovie({ _meta: { createdBy } });
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.ts]: 👍

File: libs/movie/src/lib/movie/+state/movie.model.ts [hunk 4]
```diff
@@ -109,9 +109,10 @@ export function createMoviePromotionalElements(
   return {
     trailer: [],
     still_photo: [],
+    poster: [],
     ...params,
     // We want a default poster as we look for the first one
-    poster: params.poster.length ? params.poster : [createPromotionalElement()],
+/*     poster: params.poster.length ? params.poster : [createPromotionalElement()], */
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/+state/movie.model.ts]: Remove that. You're creating bugs on others pages by doing that.
Either wait for Bruce's code to be merge or do : 
`poster: (params.poster && params.poster.length) ? params.poster : [createPromotionalElement()]`

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.html [hunk 5]
```diff
@@ -9,20 +9,26 @@ <h1 subtitle>Credits</h1>
...
         <mat-divider></mat-divider>
-        <h6 class="mat-body-2">Co-Production Company</h6>
+        <h6 class="mat-body-2">
+          Co-Production Compan{ main.get('stakeholders').get('coProducer').controls.length, plural, =1 {y} other {ies}}
+        </h6>
         <movie-form-stakeholders [form]="main.get('stakeholders').get('coProducer')"></movie-form-stakeholders>
         <mat-divider></mat-divider>
-        <p class="mat-body-2">Producers Name</p>
-        <movie-form-sales-cast role="producer" [form]="salesCast.get('producers')"></movie-form-sales-cast>
+        <p class="mat-body-2">Producer{ salesCast.get('producers').controls.length, plural, =1 {} other {s}}</p>
```
[the remark this answers is not in the document]
Comment (Maxstgt) [reply apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.html]: as you can see, I didnt changed the tags.

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.html (hunk 1 above)
Comment (Maxstgt) [reply apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.html]: its a blue version of it

**Pull request merged**